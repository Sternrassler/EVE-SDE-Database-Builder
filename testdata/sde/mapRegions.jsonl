{"regionID":10000001,"regionName":"Derelik","x":4.5e16,"y":2.0e16,"z":-1.8e17,"xMin":4.2e16,"xMax":4.8e16,"yMin":1.7e16,"yMax":2.3e16,"zMin":-2.0e17,"zMax":-1.6e17,"factionID":null,"nebula":null,"radius":3.5e16,"descriptionID":null}
{"regionID":10000002,"regionName":"The Forge","x":1.2e17,"y":5.0e16,"z":-3.0e16,"xMin":1.1e17,"xMax":1.3e17,"yMin":4.5e16,"yMax":5.5e16,"zMin":-4.0e16,"zMax":-2.0e16,"factionID":500001,"nebula":null,"radius":4.0e16,"descriptionID":null}
{"regionID":10000003,"regionName":"Vale of the Silent","x":-8.0e16,"y":1.5e17,"z":5.0e16,"xMin":-9.0e16,"xMax":-7.0e16,"yMin":1.4e17,"yMax":1.6e17,"zMin":4.0e16,"zMax":6.0e16,"factionID":null,"nebula":1,"radius":3.0e16,"descriptionID":null}
