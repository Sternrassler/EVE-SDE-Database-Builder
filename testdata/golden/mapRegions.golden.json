[
  {
    "regionID": 10000001,
    "regionName": "Derelik",
    "x": 45000000000000000,
    "y": 20000000000000000,
    "z": -180000000000000000,
    "factionID": null
  },
  {
    "regionID": 10000002,
    "regionName": "The Forge",
    "x": 120000000000000000,
    "y": 50000000000000000,
    "z": -30000000000000000,
    "factionID": 500001
  },
  {
    "regionID": 10000003,
    "regionName": "Vale of the Silent",
    "x": -80000000000000000,
    "y": 150000000000000000,
    "z": 50000000000000000,
    "factionID": null
  }
]